public/index.php

```diff
@@ -21,33 +21,41 @@
 // Create first the request object (to avoid other classes reading from php://input specially)
 $request = Request::createFromGlobals();
 
-ini_set('display_errors', 0);
 error_reporting(E_ALL & ~E_NOTICE & ~E_USER_DEPRECATED); // for symfony user deprecated errors
+
+// Bored? Try the hard way and fix some notices:
+//Symfony\Component\Debug\Debug::enable();
 // error handle needs to go after autoload
 set_error_handler('Goteo\Application\App::errorHandler');
 
-// Config file...
 $config = getenv('GOTEO_CONFIG_FILE');
 if(!is_file($config)) $config = __DIR__ . '/../config/settings.yml';
+
 Config::load($config);
 Config::autosave();
 
-// Due a symfony issue, disable FORWARDED header, it may cause some problems
-// if not exactly the same as the X_FORWARDED_FOR
-// See https://stackoverflow.com/questions/44543649/conflict-between-http-headers-in-symfony-3
-Request::setTrustedHeaderName(Request::HEADER_FORWARDED, null);
+if (Config::get('debug')) {
+    ini_set('display_errors', 1);
+    App::debug(true);
+}
 
-// Add trusted proxies
 if (is_array(Config::get('proxies'))) {
-    $request->setTrustedProxies(Config::get('proxies'));
+    $request->setTrustedProxies(
+        Config::get('proxies'),
+        Request::HEADER_FORWARDED
```

Comment:
```suggestion
        Request::HEADER_X_FORWARDED_ALL
```
Our servers use `X_FORWARDER_ALL`

I suspect this should come from a config value (having a default in `defaults.yml`)